client/components/theme-tier/theme-tier-badge/theme-tier-bundled-badge.js

```diff
@@ -1,82 +1,42 @@
-import { PLAN_BUSINESS, getPlan } from '@automattic/calypso-products';
-import { BundledBadge, PremiumBadge } from '@automattic/components';
-import { createInterpolateElement } from '@wordpress/element';
-import clsx from 'clsx';
-import { useTranslate } from 'i18n-calypso';
+import { BundledBadge } from '@automattic/components';
 import useIsUpdatedBadgeDesign from 'calypso/landing/stepper/declarative-flow/internals/steps-repository/design-setup/hooks/use-is-updated-badge-design';
 import { useBundleSettingsByTheme } from 'calypso/my-sites/theme/hooks/use-bundle-settings';
 import { useSelector } from 'calypso/state';
 import { canUseTheme } from 'calypso/state/themes/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
-import ThemeTierBadgeCheckoutLink from './theme-tier-badge-checkout-link';
 import { useThemeTierBadgeContext } from './theme-tier-badge-context';
-import ThemeTierTooltipTracker from './theme-tier-tooltip-tracker';
+import ThemeTierIncludedBadge from './theme-tier-included-badge';
+import ThemeTierUpgradeBadge from './theme-tier-upgrade-badge';
 
-export default function ThemeTierBundledBadge() {
-	const translate = useTranslate();
+export default function ThemeTierBundledBadge( { hideBackgroundOnUpgrade, hideBundledBadge } ) {
 	const siteId = useSelector( getSelectedSiteId );
 	const { showUpgradeBadge, themeId } = useThemeTierBadgeContext();
+
 	const bundleSettings = useBundleSettingsByTheme( themeId );
 	const isThemeIncluded = useSelector(
-		( state ) => siteId && canUseTheme( state, siteId, themeId )
+		( state ) => siteId && canUseTheme( state, siteId, themeId ),
+		( prev, next ) => prev === next
```

Comment:
Nitpick (non-blocking): This is unnecessary, since it's the default behavior
```suggestion
		( state ) => siteId && canUseTheme( state, siteId, themeId )
```